Dockerfile

```diff
@@ -4,7 +4,25 @@ FROM varnish:${VERSION}
 
 USER root
 
-RUN apt-get update && apt-get install -y prometheus-varnish-exporter &&\
- rm -rf /var/lib/apt/lists/*
+RUN set -e; \
+    apt-get update; \
+    apt-get -y install prometheus-varnish-exporter libmaxminddb-dev $VMOD_DEPS /pkgs/*.deb;
```

Comment:
libmaxminddb-dev... is there a -prod version 😅 ?

Reply:
No 😄 it's a requirement to build [libvmod-geoip2](https://github.com/varnishcache-friends/libvmod-geoip2)